WordPress/Classes/ViewRelated/Post/WPPostViewController.m

```diff
@@ -209,6 +210,8 @@ - (instancetype)initWithPost:(AbstractPost *)post
     BOOL changeToEditModeDueToUnsavedChanges = (mode == kWPEditorViewControllerModePreview
                                                 && [post hasUnsavedChanges]);
     
+    self.isOpenedDirectlyForEditing = (mode == kWPEditorViewControllerModeEdit);
```

Comment:
We have it as a best practice to avoid referencing `self` in an init method and instead set a properties ivar directly.  We'd like any new code added to conform to this practice even tho we still have code in places that doesn't (as you can see in this method).
Also we'd want to set the ivar from within the `if` block after the call to `super initWithMode:`.